languages/common-lisp/exercises/concept/pal-picker/.docs/instructions.md

```diff
@@ -54,7 +54,7 @@ The habitats needed by each size pet are:
 ## 3. And Now, We Feast
 
 One thing all earthling pets have in common is their need for food! This concept
-is somewhat alien to Ludwig, however, as he is prone to forgetting to refill his
+is somewhat alien to Ludwig, however, as they are prone to forgetting to refill their
 pet's food-bowl.
 
 Ludwig could use a simple function called `feeding-time-p` to alert him when the
```

Comment:
So far as I can tell, the "him" here is the last one! Then I'm happy to merge!

Reply:
OK, squashed this one.

This was supposed to be an "easy" issue to fix! I should  have known better :).

Once you approve (assuming you don't find any more issues) I'll merge.